bin/index.js

```diff
@@ -45,6 +45,16 @@ config(function (versions) {
   }
 
   async.doWhilst(getBin, whileCondition, function () {
-    if (switcherBin) return switcher(switcherBin)
+    if (switcherBin) {
+      var switcherProcess = switcher(switcherBin)
+
+      switcherProcess.on('close', function (code) {
+        if (code === 0) {
+          console.log(maxSatisfyVersion)
```

Comment:
I understand. I'm not considering moving to n right now, but maybe in the futur.
You don't have to merge this pull request. It was just a suggestion. And it may help anyone that has this problem to solve it